content/swiftui/concepts/views/terms/textfield/textfield.md

```diff
@@ -0,0 +1,67 @@
+---
+Title: 'TextField'
+Description: 'A view that allows the user to insert information, type memos, and edit the content.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`TextField`** is a view that allows the user to fill in the field with data. 
+
+## Syntax
+
+The first parameter refers to the text that will be displayed in a lighter color inside the field. 
+The second parameter refers to what the input will be, it's a binding parameter so that swift will always know if and when this parameter changes.
```

Comment:
Can we move these to underneath the code block? Also if we can explicitly refer to the parameter names that would be great. Feel free to keep in paragraph or change to a bullet point format.